checks/dockerhost.go

```diff
@@ -162,9 +169,13 @@ func AuditLibDocker(client *client.Client) (res Result) {
 }
 
 func AuditEtcDocker(client *client.Client) (res Result) {
-	var ruleExists bool
 	res.Name = "1.9 Audit Docker files and directories - /etc/docker"
-	ruleExists = checkAuditRule("/etc/docker")
+	ruleExists, err := checkAuditRule("/etc/docker")
+	if err != nil {
```

Comment:
Maybe turn the `err` around

```golang
if err == nil {
  if ruleExists {
     res.Pass()
  } else {
     res.Fail("")
  }
}
return
```

It would be even better if it could be turned into this:
```
if err == nil {
     return res.Pass()
}

return res.Fail(err)
```